vault/logical_system.go

```diff
@@ -2069,6 +2075,10 @@ func (b *SystemBackend) handleAuthTable(ctx context.Context, req *logical.Reques
 		}
 
 		info := mountInfo(entry)
+		status, ok := b.Core.builtinRegistry.DeprecationStatus(entry.Type, consts.PluginTypeCredential)
+		if ok {
```

Comment:
Be consistent. Either always add a deprecation status (even if it's invalid), or don't.